### PR TITLE
add the rke2 images scripts to download the release artifacts (airgap)

### PR DESCRIPTION
**Pull request by alknopfler**
add the rke2 images scripts to download the release artifacts (airgap)

### CODE REVIEW

File: scripts/day2/edge-release-rke2-images.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+https://github.com/rancher/rke2/releases/download/v1.28.8%2Brke2r1/sha256sum-amd64.txt
```
Comment (atanasdinov) [inline scripts/day2/edge-release-rke2-images.txt:1]: Edge 3.0.1 is using Kubernetes 1.28.9. Should we adjust?

Comment (alknopfler) [reply scripts/day2/edge-release-rke2-images.txt:1]: yes, i will change but it's an example

Comment (alknopfler) [reply scripts/day2/edge-release-rke2-images.txt]: changed

Comment (hardys) [reply scripts/day2/edge-release-rke2-images.txt]: Note that I don't think we can say it's just an example - for day-2 we've cut a release with specific versions e.g https://github.com/suse-edge/fleet-examples/releases/tag/release-3.0.1 - we'll have to do the same for future z-streams and ensure all the files reflect the correct versions.

**Review by hardys — APPROVED** [review]

**Pull request merged**